packages/website/src/docs/language-basics/alias.md

```diff
@@ -0,0 +1,16 @@
+---
+id: aliases
+title: Aliases
+---
+
+# Alias
+
+Aliases can be defined for types. This can be helpeful to reuse a complex expression.
+
+Alias is only a syntax helper, it has no represenation in the type graph. This means that aliases cannot be decorated. If wanting to provide a new name look for [`model is`]({{"/docs/language-basics/models" | url}})
```

Comment:
```suggestion
Alias is only a syntax helper, and it has no representation in the type graph. This means that aliases cannot be decorated. Use [`model is`]({{"/docs/language-basics/models" | url}}) to provide an alternate name for a model.
```

Reply:
Although you may want to mention `@friendlyName` in this context

Reply:
Well I was really seeing this statement as telling someone that they shouldn't do `alias Foo = string` but `model Foo is string` if they want a new name